components/script/dom/compositionevent.rs

```diff
@@ -34,7 +34,7 @@ impl CompositionEvent {
     pub fn new_uninitialized(window: &Window) -> DomRoot<CompositionEvent> {
         reflect_dom_object(Box::new(CompositionEvent::new_inherited()), window)
     }
-
+    #[allow(clippy::too_many_arguments)]
```

Comment:
Please preserve the newline between the two functions:
```suggestion

    #[allow(clippy::too_many_arguments)]
```

Reply:
done, have added the newline